src/gui/DatabaseWidget.cpp

```diff
@@ -452,9 +453,21 @@ void DatabaseWidget::openUrl()
 
 void DatabaseWidget::openUrlForEntry(Entry* entry)
 {
-    if (!entry->url().isEmpty()) {
-        QDesktopServices::openUrl(entry->url());
+    QString UrlString = entry->url();
+    if (UrlString.isEmpty())
+        return;
+
+    UrlString.replace("{TITLE}", entry->title(), Qt::CaseInsensitive);
+    UrlString.replace("{USERNAME}", entry->username(), Qt::CaseInsensitive);
+    UrlString.replace("{PASSWORD}", entry->password(), Qt::CaseInsensitive);
+
+    if (UrlString.startsWith("cmd://") && UrlString.length()>6){
```

Comment:
QDesktopServices::openUrl() will be called if UrlString=="cmd://".
Better to remove the return statement and put openUrl() into an else branch.